server/routes/orderhistory.js

```diff
@@ -0,0 +1,29 @@
+import express from "express";
+import cors from 'cors';
+import db from "../db/db-connection.js";
+const router = express.Router();
+
+
+//GET REQUEST FOR ORDER HISTORY
+router.get("/", cors(),  async (req, res) => {
+    const userHistory = {
+      sub: req.query.sub,
+    }
+    console.log([userCart.sub])
+      try {
+        const historyIdSql= 'SELECT id FROM users WHERE sub= $1';
+        const his_customerInfo = await db.query(historyIdSql, [userHistory.sub])
```

Comment:
[Optional] This step should be done in your own middleware because you would want to look up your DB PK id of a user from the sub on all authenticated requests. You can then store the user id on the request in a standard variable.

Here is an example I googled:
https://auth0.com/docs/quickstart/webapp/express#5-configure-router

This should really be part of the template you all started from.